tests/test_combined_example.py

```diff
@@ -1,53 +1,35 @@
 import os
 import unittest
 from datetime import date
-from http import HTTPStatus
 from time import sleep
 
 import hydra
 from dotenv import load_dotenv
 from hydra.core.utils import configure_log
-from hydra.experimental import initialize, compose
+from pathlib import Path

```

Comment:
who is using it?

Reply:
on line 23 

`env_path = Path(os.path.join(current_path, 'test_data', 'load_env.env'))`